src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -512,7 +515,15 @@ function FilterSidebarSearchTab({
                         options={sectorOptions}
                         value={sectors}
                         onChange={updateSector}
-                        disabled={fetchingOptions || fetchingFacilities}
+                        onFocus={() =>
+                            sectorOptions.length === 0 &&
```

Comment:
I verified that have a propType which declares`sectorObtions` as required so we don't need to worry about null/undefined here 👍